### PR TITLE
Cyberark fix fetch creds

### PR DESCRIPTION
**Pull request by ShahafBenYakir**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/32894

## Description
Handled the flow where credentials should be fetched for a given creds name.

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShahafBenYakir** [comment]
Failing UT build https://app.circleci.com/pipelines/github/demisto/content/62894/workflows/e2fb84e8-f197-4090-a026-35c81b4e8e68/jobs/260189

File: Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py [hunk 1]
```diff
@@ -152,7 +157,7 @@ def main():
             'fetch-credentials': fetch_credentials
         }
         if command in commands:
-            return_results(commands[command](client))  # type: ignore[operator]
+            return_results(commands[command](client, demisto.args()))  # type: ignore[operator]
```
Comment (yaakovi) [inline Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py:160]: `list_credentials_command` and `test_module ` doesn't expect the args.

**Review by yaakovi — CHANGES_REQUESTED** [review]

File: Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py (hunk 1 above)
Comment (ShahafBenYakir) [reply Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py]: I don't know why I thought it shouldn't be a problem but it obviously is.

File: Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py [hunk 2]
```diff
@@ -151,13 +152,13 @@ def main():
...
-            'fetch-credentials': fetch_credentials
-        }
-        if command in commands:
-            return_results(commands[command](client, demisto.args()))  # type: ignore[operator]
+        if command == 'test-module':
+            return_results(test_module(client))
+        elif command == 'cyberark-aim-list-credentials':
+            return_results(list_credentials_command(client))
+        elif command == 'fetch-credentials':
+            return_results(fetch_credentials(client, demisto.args()))
```
Comment (yaakovi) [inline Packs/cyberark_AIM/Integrations/CyberArkAIM_v2/CyberArkAIM_v2.py:160]: or move the `demisto.credentials` here
```suggestion
           fetch_credentials(client, demisto.args())
```

**Review by yaakovi — APPROVED** [review]

**Pull request merged**